crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -44,6 +47,7 @@ pub async fn launch<Provider, Pool, Network, Tasks, EngineApi, EngineT, EvmConfi
     socket_addr: SocketAddr,
     secret: JwtSecret,
     evm_config: EvmConfig,
+    #[cfg(feature = "optimism")] sequencer_client: SequencerClient,
```

Comment:
we don't need this here, because the auth server does not need an instance of the client